dockerfiles/Dockerfile-mina-rosetta

```diff
@@ -19,6 +19,19 @@ ARG ROSETTA_CLI_VERSION=v0.10.1
 
 ENV DEBIAN_FRONTEND noninteractive
 
+ARG deb_suffix
+
+# if --build-arg deb_suffix is defined, add hypen at beginning.
+ENV SUFFIX=${deb_suffix:+-${deb_suffix}}
+# construct mina debian package name based on network and suffix.
+# possible values:
+# - mina-devnet
+# - mina-devnet-lightnet etc.
+ENV MINA_DEB=mina-${network}${SUFFIX}
+ENV MINA_ARCHIVE_DEB=mina-archive-${network}
+ENV MINA_ROSETTA_DEB=mina-rosetta-${network}
```

Comment:
We will need a hardfork version of archive node as well some day I think? Not sure about rosetta.

Reply:
Yes, we need hf also for them too